Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
I must admit that I can't follow this line of argument, or else I just don't agree with it.

AutoFixture already has perhaps 90% of the building blocks necessary to implement this feature, so I don't think it's warranted to add more code to the code base. That code would also be something that we'd have to maintain and support in the future.

We keep circling back to [my original proof of concept](http://blog.ploeh.dk/2013/04/08/how-to-automatically-populate-properties-with-automoq), which manages to implement the behaviour in question in less code. Why not use that approach?